megamek/src/megamek/client/ui/swing/CommonMenuBar.java

```diff
@@ -243,6 +247,16 @@ public CommonMenuBar() {
         fileGameSave.addActionListener(this);
         fileGameSave.setActionCommand(ClientGUI.FILE_GAME_SAVE);
         submenu.add(fileGameSave);
+        fileGameQSave = new JMenuItem(Messages.getString("CommonMenuBar.fileGameQuickSave")); //$NON-NLS-1$
+        fileGameQSave.addActionListener(this);
+        fileGameQSave.setActionCommand(ClientGUI.FILE_GAME_QSAVE);
+        fileGameQSave.setAccelerator(KeyStroke.getKeyStroke(KeyEvent.VK_S, InputEvent.CTRL_DOWN_MASK));
+        submenu.add(fileGameQSave);
+        fileGameQLoad = new JMenuItem(Messages.getString("CommonMenuBar.fileGameQuickSave")); //$NON-NLS-1$
```

Comment:
This should probably be QuickLoad instead.

Reply:
Yes.